map_gui/src/render/traffic_signal.rs

```diff
@@ -212,7 +222,13 @@ fn draw_time_left(
 ) {
     let radius = Distance::meters(2.0);
     let center = app.map().get_i(i).polygon.center();
-    let percent = time_left / stage.phase_type.simple_duration();
+    let duration = stage.phase_type.simple_duration();
+    let percent = time_left
```

Comment:
I think `let percent = if duration > Duration::ZERO { time_left / duration } else { 1.0 }` would be more clear